tests/unit/multiclass/LaRank_unittest.cc

```diff
@@ -6,56 +6,54 @@
 
 using namespace shogun;
 
-TEST(DISABLED_LaRank,train)
+TEST(LaRank,train)
 {
-	index_t num_vec=10;
-	index_t num_feat=3;
-	index_t num_class=num_feat; // to make data easy
-	float64_t distance=15;
+	index_t num_vec = 10;
```

Comment:
please refrain from whitespace refactorings (see this whole unittest patch)... it just makes git bisecting more complicated. just make sure that your patches are following the codestyle requirements